packages/vscode-extension/src/controls/QuickStart.scss

```diff
@@ -8,6 +8,10 @@
     .section{
         display: flex;
         margin-bottom: 36px;
+
+        .title {
+            min-width: 300px;
```

Comment:
As you stretch the page, will title div goes wider than contents?

Reply:
No, it won't.